.github/workflows/luajit-integration.yml

```diff
@@ -0,0 +1,79 @@
+name: LuaJIT integration testing
+
+on:
+  workflow_call:
+    inputs:
+      buildtype:
+        description: CMake build type value
+        required: false
+        type: string
+        default: Debug
+      GC64:
+        description: CMake option value to enable LuaJIT GC64 mode
+        required: false
+        type: string
+        default: OFF
+      host:
+        description: Type of machine to run the GitHub job on
+        required: true
+        type: string
+      release:
+        description: Git revision from tarantool/tarantool repository
+        required: false
+        type: string
+        default: master
+      revision:
+        description: Git revision from tarantool/luajit repository
+        required: true
+        type: string
+
+jobs:
+  luajit-integration:
+    runs-on: ${{ inputs.host }}
+    steps:
+      - name: checkout
```

Comment:
It would be good to capitalise step name for consistency.

Reply:
Fixed.